atlite/__init__.py

```diff
@@ -11,7 +11,7 @@
 """
 
 __author__ = (
-    "The Atlite Authors: Gorm Andresen (Aarhus University), "
+    "Gorm Andresen (Aarhus University), "
```

Comment:
```suggestion
    "Gorm Andresen, "
```

The author institutional affliliations are a hazzle to maintain, let's get rid of them at least here. All of them are incorrect as they are anyways.